.env.sample

```diff
@@ -3,5 +3,5 @@ REPOS=
 SINCE=12.months.ago.beginning_of_month
 GITHUB_LOGIN=
 GITHUB_OAUTH_TOKEN=
-TRAVIS_BRANCH_BLACKLIST={"silverstripe-labs/silverstripe-newsletter":["0.3","0.4"]}
+TRAVIS_IGNORED_REPOS=18F/save-ferris
```

Comment:
FWIW, the Team API filters out any private/ignored items from https://team-api.18f.gov/public/api/, so you shouldn't need this filtering here and throughout, I don't think.